frontend/lib/Repository/AdminRepository/groups_repository.dart

```diff
@@ -203,4 +204,58 @@ class GroupsRepository {
     }
     return Future.value(false);
   }
+
+  Future<List<UserSummary>> getMembersNotInGroup(int groupId) async {
+    final AuthResponse authResponse = await userGateway.getFromMemory();
+
+    if (authResponse.success) {
+      final String token = authResponse.tokens!.accessToken;
+      _client.options.headers["Authorization"] = "Bearer $token";
+
+      try {
+        final response = await _client.get(
+          backendURL + usersNotInGroup,
+          queryParameters: {'groupId': groupId},
+        );
+        if (response.statusCode == 200) {
+          final BackendResponse backendResponse =
+              BackendResponse.fromJson(response.data);
+          if (backendResponse.error == "") {
+            final List<UserSummary> usersList = backendResponse.data
+                .map<UserSummary>(
+                  // ignore: unnecessary_lambdas
+                  (e) => UserSummary.fromJson(e),
+                )
+                .toList();
+            return Future.value(usersList);
+          }
+        }
+      } on DioError catch (error) {
+        return Future.error(error);
+      }
+    }
+    return Future.value([]);
+  }
+
+  Future<bool> addMember(int userId, int groupId) async {
+    final AuthResponse authResponse = await userGateway.getFromMemory();
+
+    if (authResponse.success) {
+      final String token = authResponse.tokens!.accessToken;
+      _client.options.headers["Authorization"] = "Bearer $token";
+
+      try {
+        final response = await _client.post(
+          backendURL + addUserToGroupURL,
+          queryParameters: {'groupId': groupId, 'userId': userId},
+        );
+        if (response.statusCode == 200) {
+          return Future.value(true);
+        }
+      } on DioError catch (error) {
+        return Future.error(error);
+      }
+    }
+    return Future.value(false);
```

Comment:
Nie lepiej by było w tym miejscu przesłać `response`, zamiast informacji, że zapytanie się (nie)udało?

Reply:
Wiesz co, mamy ten fragment w wielu repo, wiec po prostu dodalabym to do listy refactoringowej